README.md

```diff
@@ -12,3 +12,21 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the span of the parallel program, in terms of worst-case $\Theta$? Hint:
 It may help to consider the DAG of the parallel program.
+
+So the mergeSortAsync function is recursivley dividing the array into halves at each step until the array reaches 1. Then at each recursive step there is $\Theta(log n)$ work because it is dividing the array into havles. The sort function is taking $n$ time where $n$ is the number of elements. So there is $\Theta(n log n)$ work being done. But the span is just $\Theta(n)$ because after all the recursive calls the sort function is called and needs the sorted halves, becuause it is not working in parallel it dominates the span.
```

Comment:
But you're calling sort more than once, aren't you?

Reply:
Yes, at each level of recursion after the recursion. So every time the sorted left and right subarrays need merged.

Reply:
Then the span can't be just n.

Reply:
Ok then it would be nlogn to account for each level. I'll change it